[2016-11-14 22:37:24] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ','' in /var/www/html/code-dev/database/seeds/DatabaseSeeder.php:17
Stack trace:
#0 {main}  
[2016-11-14 22:37:52] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'syntax error, unexpected ','' in /var/www/html/code-dev/database/seeds/DatabaseSeeder.php:17
Stack trace:
#0 {main}  
[2016-11-14 22:39:03] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:40:51] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:41:18] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:47:57] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:48:01] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:48:06] local.ERROR: exception 'InvalidArgumentException' with message 'Command "db:Seed" is not defined.

Did you mean this?
    db:seed' in /var/www/html/code-dev/vendor/symfony/console/Application.php:519
Stack trace:
#0 /var/www/html/code-dev/vendor/symfony/console/Application.php(187): Symfony\Component\Console\Application->find('db:Seed')
#1 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-11-14 22:48:17] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:51:06] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:54:30] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 22:54:38] local.ERROR: exception 'InvalidArgumentException' with message 'Command "db:Seed" is not defined.

Did you mean this?
    db:seed' in /var/www/html/code-dev/vendor/symfony/console/Application.php:519
Stack trace:
#0 /var/www/html/code-dev/vendor/symfony/console/Application.php(187): Symfony\Component\Console\Application->find('db:Seed')
#1 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#3 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#4 {main}  
[2016-11-14 22:54:54] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: App\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): App\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:20:38] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'App\Client' not found' in /var/www/html/code-dev/database/seeds/DatabaseSeeder.php:15
Stack trace:
#0 {main}  
[2016-11-14 23:21:16] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(15): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(15): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:21:38] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(15): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(15): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:22:46] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:24:40] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:25:43] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(17): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:26:48] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:26:52] local.ERROR: exception 'BadMethodCallException' with message 'Call to undefined method Illuminate\Database\Query\Builder::trucante()' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Query/Builder.php:2123
Stack trace:
#0 [internal function]: Illuminate\Database\Query\Builder->__call('trucante', Array)
#1 [internal function]: Illuminate\Database\Query\Builder->trucante()
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Builder.php(1015): call_user_func_array(Array, Array)
#3 [internal function]: Illuminate\Database\Eloquent\Builder->__call('trucante', Array)
#4 [internal function]: Illuminate\Database\Eloquent\Builder->trucante()
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3444): call_user_func_array(Array, Array)
#6 [internal function]: Illuminate\Database\Eloquent\Model->__call('trucante', Array)
#7 [internal function]: CodeProject\Client->trucante()
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Eloquent/Model.php(3458): call_user_func_array(Array, Array)
#9 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Database\Eloquent\Model::__callStatic('trucante', Array)
#10 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): CodeProject\Client::trucante()
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#12 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#15 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#17 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#18 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#19 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#21 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#22 {main}  
[2016-11-14 23:44:39] local.ERROR: exception 'ErrorException' with message 'Use of undefined constant ClientTableSeeder - assumed 'ClientTableSeeder'' in /var/www/html/code-dev/database/seeds/DatabaseSeeder.php:16
Stack trace:
#0 /var/www/html/code-dev/database/seeds/DatabaseSeeder.php(16): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(8, 'Use of undefine...', '/var/www/html/c...', 16, Array)
#1 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Database/Console/Seeds/SeedCommand.php(61): DatabaseSeeder->run()
#2 [internal function]: Illuminate\Database\Console\Seeds\SeedCommand->fire()
#3 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Container/Container.php(507): call_user_func_array(Array, Array)
#4 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(150): Illuminate\Container\Container->call(Array)
#5 /var/www/html/code-dev/vendor/symfony/console/Command/Command.php(256): Illuminate\Console\Command->execute(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#6 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Console/Command.php(136): Symfony\Component\Console\Command\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#7 /var/www/html/code-dev/vendor/symfony/console/Application.php(846): Illuminate\Console\Command->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#8 /var/www/html/code-dev/vendor/symfony/console/Application.php(190): Symfony\Component\Console\Application->doRunCommand(Object(Illuminate\Database\Console\Seeds\SeedCommand), Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#9 /var/www/html/code-dev/vendor/symfony/console/Application.php(121): Symfony\Component\Console\Application->doRun(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#10 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Console/Kernel.php(107): Symfony\Component\Console\Application->run(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#11 /var/www/html/code-dev/artisan(36): Illuminate\Foundation\Console\Kernel->handle(Object(Symfony\Component\Console\Input\ArgvInput), Object(Symfony\Component\Console\Output\ConsoleOutput))
#12 {main}  
[2016-11-15 14:28:01] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:53
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#4 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#6 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#7 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#9 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#10 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 /var/www/html/code-dev/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2016-11-15 14:35:45] local.ERROR: exception 'Illuminate\Session\TokenMismatchException' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/VerifyCsrfToken.php:53
Stack trace:
#0 [internal function]: Illuminate\Foundation\Http\Middleware\VerifyCsrfToken->handle(Object(Illuminate\Http\Request), Object(Closure))
#1 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#2 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#3 [internal function]: Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#4 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#5 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(62): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#6 [internal function]: Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#7 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#8 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#9 [internal function]: Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#10 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#11 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(59): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#12 [internal function]: Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#13 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#14 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/CheckForMaintenanceMode.php(44): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#15 [internal function]: Illuminate\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#16 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(124): call_user_func_array(Array, Array)
#17 [internal function]: Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#18 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(103): call_user_func(Object(Closure), Object(Illuminate\Http\Request))
#19 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(122): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#20 /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(87): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#21 /var/www/html/code-dev/public/index.php(54): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#22 {main}  
[2016-11-15 14:43:40] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'CodeProject\Http\Controllers\Client' not found' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:40
Stack trace:
#0 {main}  
[2016-11-15 15:10:17] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-15 15:12:01] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-15 15:15:06] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-15 15:15:32] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-15 15:16:25] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-15 15:18:06] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Call to a member function delete() on null' in /var/www/html/code-dev/app/Http/Controllers/ClientController.php:86
Stack trace:
#0 {main}  
[2016-11-16 02:25:16] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 02:28:24] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 02:29:25] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 02:32:43] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 02:37:30] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 17:58:58] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
[2016-11-16 18:46:05] local.ERROR: exception 'Symfony\Component\Debug\Exception\FatalErrorException' with message 'Class 'Prettus\Repository\Providers\RepositoryServiceProvider' not found' in /var/www/html/code-dev/vendor/laravel/framework/src/Illuminate/Foundation/ProviderRepository.php:146
Stack trace:
#0 {main}  
